pkg/lib/superuser.js

```diff
@@ -90,13 +90,11 @@ function Superuser() {
         return (proxy.Bridges?.length ?? 0) > 0;
     };
 
-    const self = {
+    const self = cockpit.event_target({
```

Comment:
Does the return of `cockpit.event_target` provide the same object we sent in? Otherwise this is different from before

Reply:
Yes, it does.